packages/experiments/src/components/Shimmer/ShimmerCircle/ShimmerCircle.styles.ts

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import {
+  IStyle,
+  DefaultPalette,
+  IStyleSet
+} from '../../../Styling';
+import { IStyleFunction } from '../../../Utilities';
+
+export interface IShimmerCircle {
+
+}
+
+/**
+ * ShimmerCircle component props.
+ */
+export interface IShimmerCircleProps extends React.AllHTMLAttributes<HTMLElement> {
```

Comment:
Same here.

Reply:
Moved them in different file ShimmerCircle.types.ts

Reply:
Moved in a different file.